data_structures/BitManipulation/Different_Bits_Sum_Pairwise.py

```diff
@@ -0,0 +1,58 @@
+'''
+Problem Description
+We define f(X, Y) as number of different corresponding bits in binary representation of X and Y. For example, f(2, 7) = 2, since binary representation of 2 and 7 are 010 and 111,respectively. The first and the third bit differ, so f(2, 7) = 2. You are given an array of N positive integers, A1, A2 ,..., AN. Find sum of f(Ai, Aj) for all pairs (i, j) such that 1 ≤ i, j ≤ N. Return the answer modulo 109+7.       
+
+
+Problem Constraints
+  
+1 <= N <= 100000
+1 <= A[i] <= INTMAX
+       
+
+
+Input Format
+First and only argument of input contains a single integer array A.
+
+
+Output Format
+Return a single integer denoting the sum.
+
+
+Example Input
+Input 1:
+A = [1, 3, 5]
+       
+
+
+Example Output
+Ouptut 1:
```

Comment:
```suggestion
Output 1:
```
https://github.com/TheAlgorithms/Python/pull/1845/checks?check_run_id=573851978#step:5:10